.gitlab-ci.yml

```diff
@@ -1,6 +1,7 @@
 stages:
   - build
   - test
+  - gencode
```

Comment:
Can you name it "gen-code"? Otherwise it can also be read as "g-encode".

Reply:
A fair point. Renamed it.